src/cfb_api/routes/images.py

```diff
@@ -0,0 +1,46 @@
+from flask import Blueprint, request, send_file, jsonify
+from io import BytesIO
+from src.cfb_api.services.s3_service import get_image
+from botocore.exceptions import ClientError
+
+bp = Blueprint('images', __name__)
+
+@bp.route('/images', methods=['GET'])
+def get_image_from_s3():
```

Comment:
Can we add some tests for this method? We should use the pytest library and put them in the `test` directory so it has a matching folder structure to the `src` directory.